### ISSUE TITLE
"Not Connected to local network." in version 1.3.0

### ISSUE THREAD

**Issue by Subway07** [rivafarabi/deckboard#149]
Hello, I installed the DeckBoard program today, I made my boards and all of a sudden it crashed ...
When I raise, nothing (Screen shoot)
So I got to know how to do: I uninstall and reinstall, I look for tutorial on the internet and I saw that the problem is set in version 1.2.642 while I am in 1.3.0.
Moreover when I go to export, I see that my scenes are taken into account.
Do you have any worries? In addition I love the software I would be so disappointed more can use it ^^

Sorry for my english it's not my language ?

https://image.noelshack.com/fichiers/2019/14/5/1554501291-pb-1.jpg
https://image.noelshack.com/fichiers/2019/14/5/1554501298-pb-2.jpg

**Comment by Subway07** [comment]
I managed to find after 5 hours non stop .. !!
For people who have the same problem as me:
- Quit the software
- Disconnect internet
- Relancer and it's good

**Comment by nothingiscool** [comment]
I'm having the exact same problem. I tried the above workaround (quit, disconnect, reconnect), and it didn't work. There aren't any error messages or anything; it just won't connect.

**Comment by LigerXT5** [comment]
@nothingiscool I had the issue for a few days. Found out my VPN adblocker was causing issues for deckboard. I had to clear all custom settings for deckboard, basically treating it like a new app, and all is well now.

Outside of that, make sure your windows firewall and AV (some av act like a second firewall) allow the app in and outbound, and that your app and phone are on the same network and can see each other. The easier and accurate way is to find the IP of your phone (check your about phone and under wifi IP, or look at current wifi connection advanced details), then open command prompt on your computer and do: ping IP.Of.Phone.Here. If you get 5 success pings, pings showing a ##ms, then your phone and PC can talk to each other. If they can't ping each to and back, then your phone is either not on the same network as your PC, or your wifi is set to separate local network from wireless, such as a guest wifi.
